modules/ROOT/pages/x12-edi/x12-edi-connector.adoc

```diff
@@ -71,7 +71,7 @@ Create schemas to describe your messages according to your implementation.
 
 === About the EDI Schema Language
 
-The X12 EDI Module uses a YAML format called ESL (for EDI Schema Language) to represent EDI schemas. Basic ESLs define the structure of EDI messages in terms of structures (transaction sets, in X12 terminology), groups (loops, in X12), segments, composites, and elements. ESLs for the X12 versions 003010, 003020, 003030, 003040, 003050, 003060, 003070, 004010, 004020, 004030, 004040, 004050, 004060, 005010, 005020, 005030, 005040, 005050, and 006020 are included. A set of HIPAA variations of base X12 transaction sets are also included.
+The X12 EDI Module uses a YAML format called ESL (for EDI Schema Language) to represent EDI schemas. Basic ESLs define the structure of EDI messages in terms of structures (transaction sets, in X12 terminology), groups (loops, in X12), segments, composites, and elements. ESLs for the all https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-x12[supported X12 versions] are included. A set of https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-hipaa[HIPAA variations] of base X12 transaction sets are also included.
```

Comment:
```suggestion
The X12 EDI Module uses a YAML format called ESL (EDI Schema Language) to represent EDI schemas. Basic ESLs define the structure of EDI messages in terms of structures (transaction sets in X12 terminology), groups (loops in X12), segments, composites, and elements. 

ESLs for all https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-x12[supported X12 versions] are included. A set of https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-hipaa[HIPAA variations] of base X12 transaction sets are also included.
```

Reply:
Those links should be xrefs. But as long as they work right now, you can update in the next iteration when you have more time for link testing, etc.